friendshipper/src/routes/servers/+page.svelte

```diff
@@ -42,6 +42,10 @@
 		}
 	};
 
+	void listen('preferences-closed', () => {
```

Comment:
What does this fn do, how is it used if it's not called directly on this page?

Reply:
The preferences modal itself emits a `preferences-closed` event, which this component listens to.